conduit/data/datasets/vision/base.py

```diff
@@ -82,8 +84,12 @@ def _sample_x(
             index = list(range(len(self)))[index]
         if isinstance(index, list):
             sample_ls = [self._sample_x(index=i, coerce_to_tensor=coerce_to_tensor) for i in index]
-
-            if is_tensor_list(sample_ls):
+            elem = sample_ls[0]
+            if isinstance(elem, Addable):
+                summed = reduce(operator.add, sample_ls)
```

Comment:
Why not use `sum()`?

Reply:
Because sum initialises the summand as 0 and thus requires any class implementing ``__add__`` to be addable to integers.

Reply:
Oh, I didn't know.

Reply:
It seems you could do this:

```python
sum(a[1:], a[0])
```

Reply:
Yeah, that works but summing over a list doesn't -- I experimented with this quite a bit before stumbling on the reduce+operator solution.